### PR TITLE
Mention users might need to full restart Docker Desktop to have git properly detected for devenvs

### PR DESCRIPTION
**Pull request by gtardif**
Signed-off-by: Guillaume Tardif <guillaume.tardif@gmail.com>

### CODE REVIEW

File: desktop/dev-environments.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ If it doesn't detect Git as a valid command, you must reinstall Git and ensure y
 
 ![Windows add Git to path](images/dev-env-gitbash.png){:width="300px"}
 
+> **Note**
+>
+> You might need to quit and restart entirely Docker Desktop once git has been installed. From the Docker menu, select **Quit Docker Desktop**, then restart it.
```
Comment (glours) [inline desktop/dev-environments.md:48]: I don't think we need the `might` here, people must restart Docker Desktop to make the detection work properly, no? 

```suggestion
> You need to quit and restart entirely Docker Desktop once git has been installed. From the Docker menu, select **Quit Docker Desktop**, then restart it.
```

Comment (rumpl) [inline desktop/dev-environments.md:48]:
```suggestion
> You might need to quit and restart entirely Docker Desktop once git has been installed. From the Docker menu, select **Restart**, then restart it.
```

Not sure if it's "Restart" on windows too

Comment (felipecruz91) [reply desktop/dev-environments.md:48]: Just checked - it's `Restart Docker...` on Windows

Comment (gtardif) [reply desktop/dev-environments.md:48]: Actually no, "Restart Docker..." will continue execute in the same process and not catch new entries added in the PATH after Docker Desktop has been started. (Checked, the Dashboard window gets refreshed but stays alive when clicking "Restart Docker")

Comment (usha-mandya) [reply desktop/dev-environments.md]: How about "After you have installed Git, you must quit and then start Docker Desktop. From the Docker menu, select **Quit Docker Desktop**, and then start the application."?

**Review by usha-mandya — APPROVED** [review]
LGTM. Thank you

**Pull request merged**